src/commands/a32nx/freetext.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'freetext',
+    description: 'Provides a link to the FlyByWire free text tutorial video',
+    category: CommandCategory.A32NX,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the Free Text feature?',
+        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX Free Text feature [here.](https://www.youtube.com/watch?v=xED3mXqzxTc)',
```

Comment:
```suggestion
        description: 'Please watch our [video tutorial](https://www.youtube.com/watch?v=xED3mXqzxTc) on how to use the FlyByWire A32NX Free Text feature'.
```

Reply:
Replaced YouTube link with docs link as the YouTube video is no longer current.

Reply:
```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX Free Text feature [here.](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/freetext)',
```
Please use this link instead as we are / have created a dedicated feature page for this on docs. 

Relevant PR - https://github.com/flybywiresim/docs/pull/284